dev/plugins/hu.elte.txtuml.export.plantuml/src/hu/elte/txtuml/export/plantuml/seqdiag/SequenceExporter.java

```diff
@@ -53,22 +45,13 @@ public boolean preNext(MethodInvocation curElement) {
 		Expression signal = (Expression) curElement.arguments().get(1);
 		String signalExpr = signal.resolveTypeBinding().getQualifiedName();
 
-		compiler.activateLifeline(targetName);
-
 		compiler.println(senderName + "->" + targetName + " : " + signalExpr);
-
+		compiler.activateLifeline(targetName);
```

Comment:
I think we should activate the sender too, because it has to be active to send a signal.